src/azure/azureController.ts

```diff
@@ -239,6 +251,30 @@ export class AzureController {
 		}
 	}
 
+	/**
+	 * Returns Azure subscriptions with tenant and token for each given account
```

Comment:
comment doesn't match the method name, also in the method body, you used subscriptions but the type is session, Subscription == Session?